### ISSUE TITLE
Canvas resized each time setting up scope or project

### ISSUE THREAD

**Issue by machow** [paperjs/paper.js#1160]
I might be missing something here, but every time I set up a scope or create a new project, the canvas used for the view grows by a couple pixels...

[jsfiddle](https://jsfiddle.net/machow/jnmy69qr/1/)

**Comment by sapics** [comment]
canvas1 and canvas2 have same size in Google Chrome and Firefox in Windows 10.
Which browser do you check?

**Comment by machow** [comment]
What version of Chrome did you use? I see it on Version 52.0.2743.116 (64-bit). The first canvas is has height and width set at 50px, via the CSS canvas selector. The second has that style overridden by having height and width set at 52px inline.

**Comment by sapics** [comment]
My version is v53 64bit.
I guess that it might depends on box-sizing.

Below is Developer tools information.
![default](https://cloud.githubusercontent.com/assets/11169954/18422387/498c658e-78d5-11e6-9b77-533d25740cee.png)

**Comment by machow** [comment]
Huh, I just updated to v53, and still see it. Below is a screenshot of canvas2, the one with a scope setup, showing the inline styles added...

![image](https://cloud.githubusercontent.com/assets/2574498/18422711/1557169c-786c-11e6-9117-b7f37bdf4747.png)

It looks like these [lines in the source code](https://github.com/paperjs/paper.js/blob/79661c35e6d623c4598ec6dea8c4a7ae8c0c31a4/src/view/View.js#L44-L59) might set the other inline styles shown in the image.

**Comment by sapics** [comment]
It's strange :)
This is my screenshot for second one.
![2](https://cloud.githubusercontent.com/assets/11169954/18422929/4abada66-78db-11e6-866d-ca93a6fa9561.png)

**Comment by sapics** [comment]
How about this [fiddle](https://jsfiddle.net/jnmy69qr/2/)?
I guess that both of width should be 50 which includes border.

**Comment by machow** [comment]
That one works, but still sets the height and width inline in my browser..

![image](https://cloud.githubusercontent.com/assets/2574498/18423060/623f14de-786f-11e6-9777-8f6799fbc73e.png)

In any event, it seems like a reasonable way to work around the issue (or non-issue!)--thanks for taking the time to look at it. I'm verified it on another laptop with the original fiddle (also Google Chrome v52), and am happy to look at other things if the behavior is unexpected.

**Comment by sapics** [comment]
You are welcome!
It might come from some extensions in your Chrome or some special setting in about:config.

Just my two cents,
```html
<div class="border-wrapper">
  <canvas></canvas>
</div>
```
would be better for less trouble :)

**Comment by lehni** [comment]
I am seeing this too, on Mac. @machow are you on Mac? Retina screen?

**Comment by lehni** [comment]
BTW, this happens on Safari and Firefox also. It's probably because you're adding a border to the canvas directly. That's not supported at the moment. A simple workaround is to add the border to the parent container instead. Just wrap the canvas in a div.

**Comment by machow** [comment]
Yeah--Mac with a retina screen. That's really useful to know! I went crazy for a bit trying to figure out what I had done differently to cause the behavior. Wrapping in a div seems very reasonable. I left it open in case it's something you'd like to change down the road, but given the ease of working around the problem it doesn't seem like much of an issue.
